cluster/gce/gci/configure-helper.sh

```diff
@@ -51,6 +51,7 @@ function main() {
   KUBE_HOME="/home/kubernetes"
   CONTAINERIZED_MOUNTER_HOME="${KUBE_HOME}/containerized_mounter"
   PV_RECYCLER_OVERRIDE_TEMPLATE="${KUBE_HOME}/kube-manifests/kubernetes/pv-recycler-template.yaml"
+  DEFAULT_NETWORK_TEMPLATE="${KUBE_HOME}/kube-manifests/kubernetes/default-network.tmpl"
```

Comment:
looks like all usage of network template is under ${KUBE_HOME}/network.tmpl, what is the purpose for this one?

Reply:
If I first attempted to use ${KUBE_HOME}/network.tmpl directly in manifest file cluster/gce/manifests/kube-controller-manager.manifest, I failed. Using this way is working.

Reply:
When all codes are working, I try the idea again and this time the idea works. I will send new commit very soon.